server/util/perms/perms.go

```diff
@@ -290,3 +290,23 @@ func AuthenticatedGroupID(ctx context.Context, env environment.Env) (string, err
 func IsAnonymousUserError(err error) bool {
 	return status.IsUnauthenticatedError(err) || status.IsPermissionDeniedError(err) || status.IsUnimplementedError(err)
 }
+
+func GetFromContext(ctx context.Context, env environment.Env) (*UserGroupPerm, error) {
```

Comment:
The name `GetFromContext` feels a bit too general since there might be cases where it makes sense to set other perms that are derived from context, like USER_READ|USER_WRITE etc.

wdyt about renaming this to "ForAuthenticatedGroup", and adding a comment like

`ForAuthenticatedGroup returns GROUP_READ|GROUP_WRITE permissions for the authenticated group, or OTHERS_READ for anonymous users.`

Reply:
Ack - done